go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	github.com/sergi/go-diff v1.2.0
 	go.uber.org/atomic v1.10.0
 	gopkg.in/yaml.v2 v2.4.0
+	gopkg.in/yaml.v3 v3.0.1
```

Comment:
Do we need two major versions of the yaml library?

Reply:
Yes, absolutely. Both versions are in active use across our codebase (both by us and by other libraries), so we need to test that regexpext works with both of them.